src/SymphonyOSS.RestApiClient/Generated/OpenApi/AgentApi/AgentApi.cs

```diff
@@ -6543,7 +6733,7 @@ public override object ReadJson(Newtonsoft.Json.JsonReader reader, System.Type o
     
         private System.Type GetObjectSubtype(System.Type objectType, string discriminator)
         {
-            foreach (var type in System.Reflection.CustomAttributeExtensions.GetCustomAttributes<JsonInheritanceAttribute>(System.Reflection.IntrospectionExtensions.GetTypeInfo(objectType), false))
+            foreach (var type in System.Reflection.CustomAttributeExtensions.GetCustomAttributes<JsonInheritanceAttribute>(System.Reflection.IntrospectionExtensions.GetTypeInfo(objectType), true))
```

Comment:
We should verify where this change is coming from as well since we seem to be flipping the flag here.

Reply:
https://github.com/mvu02/NJsonSchema/commit/0c733d2681005615fda7cb745d52b1cf61ac3d36